pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -254,7 +254,7 @@ func configDNSSearchName(qemuArg *api.Arg) error {
 }
 
 // returns nameservers [][]byte, searchdomains []string, error
-func GetResolvConfDetailsFromPod() ([][]byte, []string, error) {
+func GetResolvConfDetailsFromPod(subdomain string) ([][]byte, []string, error) {
```

Comment:
This function is now doing at least one more thing: Adding the subdomain to the list of domains retrieved from the pod.
So in fact, it is misleading a bit... as the returned result is not the details seen in the pod.

I think this needs to be done as a second step and only on the "searchDomains".
Keeping each function doing one thing is safer and much clearer.